addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -54,6 +54,15 @@ func _init():
 		File.WRITE
 	)
 
+# Print a message
+#
+# #### Parameters
+#
+# * string: Text to log
+# * args: Additional information
+func print(string: String):
+	_log("(?)\t" + string)
```

Comment:
`_log()` function does write in the logfile, which is precisely what you want to avoid :)